pulsar-broker/src/main/java/com/yahoo/pulsar/broker/loadbalance/BrokerHostUsage.java

```diff
@@ -71,25 +91,152 @@ public BrokerHostUsage(PulsarService pulsar) {
      *
      * @throws IOException
      */
-    public String getBrokerHostUsage() throws IOException {
-        StringWriter writer = new StringWriter();
+    public synchronized SystemResourceUsage getBrokerHostUsage() throws IOException {
```

Comment:
Current implementation runs the script everytime this method is called.
How often is this method actually called?
This method would return the average usage between two calls.